docs/CHANGELOG.rst

```diff
@@ -37,6 +37,7 @@ Fixed
   ``changes-only`` mode
 - Fix parallel test execution where more test processes than databases were
   created during tests
+- Calculate ``Data.size`` by summing ``total_size`` of all file-type outputs
```

Comment:
Be careful, you wrongly placed this CHANGELOG entry under a released version of Resolwe.